modules/tests/test_opendaq_device_modules/test_streaming.cpp

```diff
@@ -443,7 +460,11 @@ INSTANTIATE_TEST_SUITE_P(
         std::make_tuple("openDAQ Native Streaming", "opendaq_native_streaming", "daq.ns://127.0.0.1/"),
         std::make_tuple("openDAQ Native Streaming", "opendaq_native_streaming", "daq.opcua://127.0.0.1/"),
         std::make_tuple("openDAQ LT Streaming", "opendaq_lt_streaming", "daq.lt://127.0.0.1/"),
-        std::make_tuple("openDAQ LT Streaming", "opendaq_lt_streaming", "daq.opcua://127.0.0.1/")
+        std::make_tuple("openDAQ LT Streaming", "opendaq_lt_streaming", "daq.opcua://127.0.0.1/"),
+        std::make_tuple("openDAQ Native Streaming", "opendaq_native_streaming", "daq.ns://[::1]/"),
```

Comment:
In my view, having duplicate tests that only differ in their use of the IPv6 connection method isn't really necessary. 
Having tests like `TEST_F(*ModulesTest, ConnectViaIpv6)` in other test suites should be sufficient. 
So, I would keep only the IPv4 connection methods here and in other parameterized tests below to optimize testing process.